ivadomed/postprocessing.py

```diff
@@ -38,40 +47,36 @@ def threshold_predictions(predictions, thr=0.5):
 
 
 @nifti_capable
+@binarize_with_low_threshold
 def keep_largest_object(predictions):
     """
     Keep the largest connected object from the input array (2D or 3D).
-    Note: This function only works for binary segmentation.
 
     Args:
-        predictions (array or nibabel object): Input binary segmentation. Image could be 2D or 3D.
```

Comment:
Technically-speaking this function does *only* accept binary inputs. The decorator (which is not part of the function) makes sure that input is indeed binary. However this is a blurry zone as when called outside, we could input non-binary (thanks to the decorator)... @kousu what is the proper way of documenting this docstring?

Reply:
Mmm I see. If we go this way, we should change `array or nibabel object` --> `array`.
But I thought it would be informative to make clear in the docstring that we could input a soft nibabel.

Reply:
> Mmm I see. If we go this way, we should change array or nibabel object --> array.

indeed, good point. 

> But I thought it would be informative to make clear in the docstring that we could input a soft nibabel.

yup, agreed. Let's wait for @kousu 's feedback.

Reply:
(Opening this issue #207, as I am about to merge this PR)